src/components/Login/LoginFeatures.js

```diff
@@ -34,45 +35,48 @@ const LoginFeatures = (props) => {
         }}
       />
       <Button value="Login" size="large" />
```

Comment:
This button is the reason for the larger gap between the password and the *real* Login button.  It needs to be removed.